mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java

```diff
@@ -37,6 +41,24 @@
 
     KYCContract.KYCDescriptionPresenter mKYCDescriptionPresenter;
 
+    @BindView(R.id.img1)
+    ImageView img1;
+
+    @BindView(R.id.img2)
+    ImageView img2;
+
+    @BindView(R.id.img3)
+    ImageView img3;
+
+    @BindView(R.id.first)
+    LinearLayout f;
+
+    @BindView(R.id.second)
+    LinearLayout s;
+
+    @BindView(R.id.third)
+    LinearLayout t;
+
```

Comment:
What are those, excuse me? It's not your local project, but global open-source app where you must keep the code style to the adopted standards

Reply:
@letelete  sorry, did not understand where did I break the code style

Reply:
Setting proper names to the fields in the classes are really important. Setting name like 't' is completely awful and it shouldn't have a place here. 

**One of the most important thing in the professional programming is to make code understandable for other developers.**

So instead of `R.id.third` it should look like `R.id.ll_level_3`

How would you know what does the `third` stands for after seeing it again like 3 weeks later? It's just a bad code.

Take a look at the other files to realize we are using name convention for the layout objects.

So instead of:

`t`

you should call it like:

`llLevel3`